test_cases/encoding.json

```diff
@@ -23,18 +23,21 @@
     },
     {
       "id": 2,
-      "status": "pass",
+      "status": "fail",
       "endpoint": "search",
       "description": "make sure that results return the appropriate accents on results",
-      "notes": "https://github.com/pelias/api/issues/731",
+      "issue": [
+        "https://github.com/pelias/api/issues/731",
+        "https://github.com/pelias/placeholder/issues/55"
+      ],
       "in": {
         "text": "pärnu, estonia"
       },
       "expected": {
         "properties": [
           {
             "name": "Parnu",
-            "region": "Pärnumaa",
+            "region": "Pärnu County",
```

Comment:
we shouldn't mix the english names with those special characters. if we selected this test because we wanted to see appropriate accents in the results then it should be ok to add the language parameter that exposes the accents in the results.

Reply:
Good suggestion. I've changed the output language to Estonia, and created another acceptance test to track the related sorting issue.